### ISSUE TITLE
Cannot use nsxt_policy_vtep_ha_host_switch_profile in nsxt_policy_host_transport_node_profile

### ISSUE THREAD

**Issue by martinrohrbach** [vmware/terraform-provider-nsxt#1441]
### Describe the bug

We have adopted the nsxt_policy_host_transport_node_profile in our environment and after some initial problems (see my last issues ;) we are quite happy with that.

We are now looking into using the nsxt_policy_vtep_ha_host_switch_profile for our transport nodes. We've tested the profiles manually in our test environment by adding the VTEP HA profiles using the API and that worked fine. However, when we tried using the vTEP HA profile in the nsxt_policy_host_transport_node_profile resource, the provider fails:

```
│
│ Error:  Failed to update Policy Host Transport Node Profile 199f39ef-9406-4a69-b9a4-6c8876573ea5: Unable to find UplinkProfile associated with Id BaseHostSwitchProfile/ac3d4c95-7500-4062-8dd0-b0e2233908cd. (code 8500)
│
│   with nsxt_policy_host_transport_node_profile.trf-tnp["cluster1"],
│   on [transport-node-profiles.tf](http://transport-node-profiles.tf/) line 109, in resource "nsxt_policy_host_transport_node_profile" "trf-tnp":
│  109: resource "nsxt_policy_host_transport_node_profile" "trf-tnp" {
│
```

We've simply added the profile to the host_switch_profile property of the resource, but apparently that expects only "BaseHostSwitchProfiles" rather than VTEP HA ones. 

As I currently don't see a way to add the latter, is this possibly something that can easily be added (either by accepting them in host_switch_profile or by adding an additional property)?

Or is it already possible and we overlooked something?

### Reproduction steps

1. Create nsxt_policy_vtep_ha_host_switch_profile resource
2. Use the resouce as a profile for nsxt_policy_host_transport_node_profile.host_switch_profile
3. Provider fails to create transport node profile (see above)

### Expected behavior

We can use the VTEP HA profile to create a host transport profile.

**Comment by ksamoray** [comment]
Hi @martinrohrbach,
Can you please include the following info:

- NSX, provider versions
- Output of the `terraform apply` execution while the following flags are set:
```
export TF_LOG=INFO
export TF_LOG_PROVIDER_NSX_HTTP=1
```

**Comment by martinrohrbach** [comment]
Fix confirmed in 3.71, many thanks!

**Comment by ksamoray** [comment]
@martinrohrbach YW, thanks for reporting this!
